reference/6/About/about_Special_Characters.md

```diff
@@ -112,6 +133,20 @@ The output from this command is:
 
 Column1         Column2         Column3
 
+UNICODE CHARACTER (`u{x})
+The Unicode escape sequence allows you to specify any Unicode character
+including Unicode surrogate characters often used for Emoji e.g. `u{1F44D}.
```

Comment:
Check on lower case emoji. RE surrogate character issue, that's probably too much detail.  How about something like this?

> specify any Unicode character including those above the Basic Multilingual Plane (> 0xFFFF) which includes emoji characters e.g. ...